kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -361,25 +364,25 @@ define([
                             if (type === 'display') {
                                 const isFolder = data === 'true' ? true : false;
                                 const icon = isFolder ? 'folder' : 'file-o';
-                                let disp = '<span><i class="fa fa-' + icon + '"></i></span>';
+                                const disp = '<span class="fa fa-' + icon + '"></span>';
```

Comment:
This file had a lot of extra wrappers around font awesome icons. I standardised them all to be of the form `<span class="fa ..."></span>`.